### DISCUSSION TITLE
OpenSuse instructions

### DISCUSSION THREAD

**Discussion by DutchPete** [Ulauncher/Ulauncher#1343] [Packaging]
For 2 days I looked for a way to install Ulauncher 5.15.6 on openSuse Leap 15.5 without having to go through the build process. [Here](https://software.opensuse.org/download/package?package=ulauncher&project=home%3ADead_Mozay) is a simple way to do so on Tumbleweed, Leap and Slowroll.

Might be useful to show on your website's home page, it would have saved me 2 days of googling.

**Comment by friday** [comment]
Thanks. I didn't know about this third party package. Without having a way to contact [Dead_Mozay](https://build.opensuse.org/project/show/home:Dead_Mozay) and knowing that it will update regularly etc I'm not sure about recommending this officially. Will have to discuss it with the other maintainers.

But we should at least link to https://github.com/Ulauncher/Ulauncher/discussions/categories/packaging for people on other distros, so they can find this thread.

I tried to use Open Build service myself but it's a nightmare to set up.

**Reply by DutchPete** [reply]
@friday thank you for your quick reply.

It took me 2 days of scouting around before I happened to stumble across the web page. I have installed ULauncher on Leap 15.5, and it works fine.

The only thing is that extensions don't work: they crash during the installation. Also, I cannot add an icon to a shortcut: when I click on the icon box and then click on the icon (I tried an svg file), nothing happens.

Should I open an issue or would that not be appropriate as my installation is from that page? Mind you, that page is an openSuse repository.

BTW, thanks for changing the title of this issue 👍

**Reply by friday** [reply]
If you have issues installing them you can report it here as an issue and I'll check it out. If they install but crash when you run them report extension issues to their issue trackers (after making sure you followed their instructions)
See https://github.com/Ulauncher/Ulauncher/issues/767 and #273

Shortcut icon issue has been reported, but I can't reproduce it: https://github.com/Ulauncher/Ulauncher/issues/1225

**Reply by DutchPete** [reply]
After they are "installed", before I run them, there is that yellow box informing me that the extension has crashed, as per the screenshot [here](https://ulauncher-extension-doesnt-install-and-now.netlify.app/). So, should I report it here or on the extension's Github page?

**Reply by friday** [reply]
Get the log and the message and try to help yourself first. It's most likely an error on your system because you don't have the dependencies. If not, then create an issue for the extension (if there isn't one already).

Run Ulauncher with `ulauncher -v` from your terminal to see the verbose log as it happens (but it should also be in ~/.local/state/ulauncher/last.log).
